packages/app-extrinsics/src/Selection.tsx

```diff
@@ -63,6 +63,16 @@ class Selection extends TxComponent<Props, State> {
             withSpinner
           />
           <Button.Or />
+          <TxButton
+            accountId={accountId}
+            isBasic
+            isDisabled={!isValid}
+            isSign
```

Comment:
Same comment as transfer - we have a lot of modals that all do submission, here we cover only a fraction of them.